tests/pytest_framework/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -81,20 +79,10 @@ def create_file_destination(self, **kwargs):
     def create_filter(self, **kwargs):
         return Filter(self.__logger_factory, **kwargs)
 
-    def create_source_group(self, drivers):
-        source_group = self.__create_statement_group("source", drivers)
-        self.__syslog_ng_config["statement_groups"].append(source_group)
-        return source_group
-
-    def create_destination_group(self, drivers):
-        destination_group = self.__create_statement_group("destination", drivers)
-        self.__syslog_ng_config["statement_groups"].append(destination_group)
-        return destination_group
-
-    def create_filter_group(self, filters):
-        filter_group = self.__create_statement_group("filter", filters)
-        self.__syslog_ng_config["statement_groups"].append(filter_group)
-        return filter_group
+    def create_group(self, drivers):
```

Comment:
Optional: Consider to call this function as: create_statement_group().
I would call the parameter not drivers, but statements (filter also can be added),
I would call the inner variable not group, but statement_group

Reply:
Agreed. I will do so. The only downside would be is that it means more typing (/longer lines) for the user. But it is a good clarification, as it is a member of the config, and statement might not be the first thing that comes to my mind when someone mentions group in config context. Also, with this pr merged, users not necessarily need to make these groups explicitely, so the longer typing is reasonable.